Conduit/Conduit.Db/User.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Conduit.Db
+{
+    public class User
+    {
+        public Guid UserId { get; set; }
+        public string  UserName { get; set; }
+        public string  Email { get; set; }
+        public string  Password { get; set; }
+        public string  Bio { get; set; }
+        public int  Age { get; set; }
+        public ICollection<Comment> Comments{ get; set; }
```

Comment:
@noor848 hmmm, thinking deeply here do we have any relation with comment? NO, so no need for this, if you need to add relation so you have to add column to comment table "CreatedBy" and be a foreign key to user table